nextbike/datapreparation/datapreparation.py

```diff
@@ -114,7 +114,7 @@ def data_preparation(df_original):
 
     return df_merged
 
-
+#Todo: 30 lines for 2 lines of logic?
```

Comment:
YES!!!
The more comments, the better someone else will understand the code in the future! :)
We should not arguee or kick out additional explanatory comments!!!!
Instead we should engage to comment more, if you ask me! :)

Reply:
Especially when kind of 3 things are done in one line, comments help to understand whats happening :)

Reply:
At this place I think it makes it a bit chaotic...